pkg/utils/topology/pd.go

```diff
@@ -23,25 +23,10 @@ func FetchPDTopology(pdClient *pd.Client) ([]PDInfo, error) {
 		return nil, err
 	}
 
-	data, err := pdClient.SendGetRequest("/members")
+	ds, err := pd.FetchMembers(pdClient)
```

Comment:
`FetchPDTopology` should be called `UnsafeFetchPDTopology` as it bypasses the target address restriction.